server/routerlicious/packages/services/src/deltaManager.ts

```diff
@@ -26,11 +30,33 @@ export class DeltaManager implements IDeltaService {
 	): Promise<ISequencedDocumentMessage[]> {
 		const baseUrl = `${this.internalAlfredUrl}`;
 		const restWrapper = await this.getBasicRestWrapper(tenantId, documentId, baseUrl);
-		const resultP = restWrapper.get<ISequencedDocumentMessage[]>(
-			`/deltas/${tenantId}/${documentId}`,
-			{ from, to, caller },
-		);
-		return resultP;
+		// if requested size > getDeltasRequestMaxOpsRange, breakdown into chunks of size getDeltasRequestMaxOpsRange
+		if (
+			to != undefined &&
+			from != undefined &&
+			to - from - 1 > this.getDeltasRequestMaxOpsRange
+		) {
+			let getDeltasFrom = from;
+			let getDeltasTo = from + this.getDeltasRequestMaxOpsRange + 1;
+			const chunkResultP: Promise<ISequencedDocumentMessage[]>[] = [];
+			while (getDeltasTo <= to && getDeltasTo - getDeltasFrom - 1 > 0) {
+				chunkResultP.push(
+					restWrapper.get<ISequencedDocumentMessage[]>(
+						`/deltas/${tenantId}/${documentId}`,
+						{ getDeltasFrom, getDeltasTo, caller },
+					),
+				);
+				getDeltasFrom = getDeltasTo - 1;
+				getDeltasTo = Math.min(to, getDeltasFrom + this.getDeltasRequestMaxOpsRange + 1);
+			}
+			return (await Promise.all(chunkResultP)).flat();
```

Comment:
We have seem request of 500,000 range, so it will ends 250 parallel requests in a short time. Will this crash the pods, will this crash the server?
can we do a 2k by 2k based loop? 
Either way, the results hold in memory will probably crash the scribe pos for memory leak(error code 134) when GC detect this.

Should we just fail the request here and have caller split request range before they call this method? (If caller need all results in one call, I think it will still crashes sribe)
Should we return just say first 10K here?